### PR TITLE
MCKIN-24508 missing translations added

### PR DESCRIPTION
**Pull request by musmanmalik**
missing translations added

### CODE REVIEW

File: drag_and_drop_v2/translations/ar/LC_MESSAGES/text.po [hunk 1]
```diff
@@ -643,3 +643,31 @@ msgstr[2] ""
 msgstr[3] ""
 msgstr[4] ""
 msgstr[5] ""
+
+#: public/js/drag_and_drop.js:1298
+msgid "You cannot add any more items to this zone."
+msgstr "لا يمكنك إضافة أي عناصر أخرى إلى هذه المنطقة."
```
Comment (wasifarbisoft) [inline drag_and_drop_v2/translations/ar/LC_MESSAGES/text.po:649]: I wonder if "." is part of arabic!
Same for other arabic translations

File: drag_and_drop_v2/translations/es_419/LC_MESSAGES/text.po [hunk 2]
```diff
@@ -473,7 +473,7 @@ msgstr "Realimentación"
 
 #: public/js/drag_and_drop.js
 msgid "Keyboard Help"
-msgstr ""
+msgstr "键盘说明"
```
Comment (wasifarbisoft) [inline drag_and_drop_v2/translations/es_419/LC_MESSAGES/text.po:476]: Is this Spanish?

**Review by wasifarbisoft — CHANGES_REQUESTED** [review]
Some minor comments!

**Comment by wasifarbisoft** [comment]
@musmanmalik ,
looks good to me!

**Review by wasifarbisoft — APPROVED** [review]

**Comment by musmanmalik** [comment]
@wasifarbisoft https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/251 a copy of this PR need to review as well.

**Pull request merged**